gopass_autotype.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import subprocess
+import sys
+import time
+from os.path import expanduser
+
+home = expanduser("~")
+my_env = os.environ.copy()
+my_env['PATH'] = '/usr/local/bin:{}'.format(my_env['PATH'])
+
+special_autotype_handlers = {
+    ":tab": lambda: do_stroke(48),
+    ":space": lambda: do_stroke(49),
+    ":enter": lambda: do_stroke(36),
+    ":delete": lambda: do_stroke(51),
+    ":delay": lambda: time.sleep(1),
+    ":clearField": lambda: clear_field_content()
+}
+
+
+def get_additional_autotype_handlers_filename():
+    base_path = os.getenv("XDG_CONFIG", home)
+    return '{base_path}/{filename}'.format(base_path=base_path, filename=".gopass_autotype_handlers.json")
+
+
+def load_additional_autotype_handlers_config():
+    handlers_path = get_additional_autotype_handlers_filename()
+    if os.path.exists(handlers_path):
+        with open(handlers_path) as f:
+            return json.load(f)
+    else:
+        return {}
```

Comment:
Not sure why we need this?

What's supposed to be in the `.gopass_autotype_handlers.json` file?

Reply:
I've added a bit of extra documentation. In the end it allows to include output from external commands in the autotype command. I explicitly use it to type YubiKey TOTP tokens in authentication forms, so I don't have to do it manually. It allows to fill forms completely, without any manual interaction (as long as you can code any part as bash commands)

Reply:
In general there is an updated documentation for the autotyping, also including the special commands and the additional handlers. Maybe that makes it more clear on how it works.